packages/@webex/plugin-meetings/src/member/member.types.ts

```diff
@@ -0,0 +1,12 @@
+export enum MediaStatus {
```

Comment:
maybe worth adding a comment that these values are coming from Locus so they cannot be changed, for example they have to be uppercase, because that's how Locus sends them?

Reply:
Yes that's correct. I'll update heading with a comment and also what each value means.

Reply:
Done